clients/backend-application-insights/package.json

```diff
@@ -30,19 +30,19 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
-    "@bentley/backend-itwin-client": "2.17.0-dev.16",
-    "@bentley/bentleyjs-core": "2.17.0-dev.16",
+    "@bentley/backend-itwin-client": "workspace:*",
+    "@bentley/bentleyjs-core": "workspace:*",
     "applicationinsights": "^1.7.5"
   },
   "peerDependencies": {
     "@bentley/itwin-client": "^2.17.0-dev.16"
```

Comment:
Let's update all peerDeps too in order to avoid changing anything in the deps section.  For that syntax I think you'll want to use `workspace:^` instead of `workspace:*` as that will replace the it with `^2.17.0-dev.6` per [the docs](https://pnpm.io/workspaces#publishing-workspace-packages)